lib/logstash/plugin_mixins/jdbc.rb

```diff
@@ -92,12 +93,20 @@ def jdbc_connect
     end
   end
 
+  private
+  def load_drivers(drivers)
+    drivers.each do |driver|
+      require driver
```

Comment:
should we wrap this with a friendly rescue block for more descriptive error logging if driver jar path is not valid?